src/ctap/storage/key.rs

```diff
@@ -73,6 +73,9 @@ make_partition! {
     // - When adding a (non-persistent) key below this message, make sure its value is bigger or
     //   equal than NUM_PERSISTENT_KEYS.
 
+    /// If this entry exists and is empty, multi-PIN is enabled.
```

Comment:
nit: I would be more precise and specify as:

> Whether multi-PIN is enabled.
>
> The value must be empty. Only presence of the value matters.

or something along those lines.

Reply:
done